lab/iam/policies/labs4.yaml

```diff
@@ -0,0 +1,33 @@
+Version: "2012-10-17"
+Statement:
+  - Effect: Allow
+    Action:
+      - fsx:CreateFileSystem
+      - fsx:DeleteFileSystem
+      - fsx:UpdateFileSystem
+      - fsx:DescribeFileSystems
+      - fsx:CreateStorageVirtualMachine
+      - fsx:DeleteStorageVirtualMachine
+      - fsx:UpdateStorageVirtualMachine
+      - fsx:DescribeStorageVirtualMachines
+      - fsx:TagResource
+      - fsx:UntagResource
+    Resource:
+      - !Sub arn:aws:fsx:${AWS::Region}:${AWS::AccountId}:file-system/*
+      - !Sub arn:aws:fsx:${AWS::Region}:${AWS::AccountId}:storage-virtual-machine/*
+  - Effect: Allow
+    Action:
+      - fsx:DescribeFileSystems
+      - fsx:DescribeStorageVirtualMachines
+      - fsx:ListTagsForResource
+    Resource:
+      - ["*"]
+  - Effect: Allow
```

Comment:
We should be able to consolidate these permissions in to the secretsmanager ones in labs1.yaml so we aren't spreading permissions for the same service over multiple files.